intdataplane/int_dataplane.go

```diff
@@ -690,14 +689,14 @@ func (d *InternalDataplane) loopUpdatingDataplane() {
 					beingThrottled = false
 				}
 				log.Info("Applying dataplane updates")
-				applyStart := monotime.Now()
+				applyStart := time.Now()
 
 				// Actually apply the changes to the dataplane.
 				d.apply()
 
 				// Record stats.
-				applyTime := monotime.Since(applyStart)
-				summaryApplyTime.Observe(applyTime.Seconds())
+				applyTime := time.Since(applyStart)
+				summaryApplyTime.Observe(time.Since(applyStart).Seconds())
```

Comment:
Do we need repeat of `time.Since(applyStart)` ? (as opposed to using `applyTime`)